src/sx/README.md

```diff
@@ -227,6 +228,40 @@ const styles = sx.create({
 });
 ```
 
+### Keyframes
+
+SX also has support for keyframes, it exports a function that handles that generates a animation name from the input you give it. You can use it like this:
+
+```jsx
+export function AnimatedComponent() {
+  return <div className={styles('text')}>text</div>;
+}
+
+const fadeIn = sx.keyframes({
+  0: {
+    opacity: 0,
+  },
+  50: {
+    opacity: 0.3,
+  },
+  100: {
+    opacity: 1,
+  },
+});
+
+const styles = sx.create({
+  text: {
+    animation: `${fadeIn} 2s`,
```

Comment:
Can you please avoid shorthand props? I know it will work but we do not recommend them so it would be nice to avoid it here. What do you think?